docs-gen/content/tools/usage.md

```diff
@@ -4,7 +4,7 @@ date: 2019-08-04T13:21:34+02:00
 weight: 2
 ---
 After installing the dependencies, as described in the
-[requirements](/tools/requirements), you can simply run:
+[requirements](/vehicle_signal_specification/tools/requirements), you can simply run:
```

Comment:
The link is fixed, but that documentations is outdated, I would suggest removing the requirments doc (or all tool docs) completely and refer to the vss-tools github

Reply:
Yes, I noticed consistencies in the online tools doc and the READMEs that I alluded to in https://github.com/GENIVI/vss-tools/issues/74. It's worth noting that the online doc containing information not in the README. Seeing rationalising the doc as a related but bigger task I stuck to fixing the broken links. I would suggest moving your good input into an enhancement ticket.

Reply:
It would have been nice if we had simply merged this simple fix first before your new PR to completely rework the tools documentation was subsequently pushed and merged. It would have saved me having to pick apart my commit and rebase for what is a straight forward PR.

Force pushed an update with the change dropped as it no longer exists upstream.